app/src/componentLibrary/attributeList/editableAttribute/attribute/attribute.scss

```diff
@@ -14,7 +14,9 @@
  * limitations under the License.
  */
 
-@mixin draw-attribute-border($width, $height, $translateX: 0, $translateY: 0) {
+@mixin draw-attribute-border($width, $height, $bgColor, $translateX: 0, $translateY: 0) {
+  background-color: $bgColor;
```

Comment:
It brakes styles, when you focused on key:value

As I can see the bg color is the same everywhere.

Reply:
Done.